src/services/feed.js

```diff
@@ -80,13 +80,6 @@ class Feed {
         .remove({ account });
     }
 
-    try {
-      // schedule resync
-      this.service('twitter').connect();
-    } catch (e) {
-      this.logger.info(e);
-    }
-
```

Comment:
why is this removed?

Reply:
this.service('twitter').connect(); is empty method. 
It was used when twitter client offered stream pushes. It is not required now, since sync timer  is started within plugin lifecycle initialization.